src/pages/ValidateLoginPage/index.website.tsx

```diff
@@ -10,51 +8,21 @@ import Navigation from '@libs/Navigation/Navigation';
 import * as Session from '@userActions/Session';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
-import {defaultProps as validateLinkDefaultProps, propTypes as validateLinkPropTypes} from './validateLinkPropTypes';
+import type {ValidateLoginPageOnyxProps, ValidateLoginPageProps} from './types';
 
-const propTypes = {
-    /** The accountID and validateCode are passed via the URL */
-    route: validateLinkPropTypes,
-
-    /** Session of currently logged in user */
-    session: PropTypes.shape({
-        /** Currently logged in user authToken */
-        authToken: PropTypes.string,
-    }),
-
-    /** The credentials of the person logging in */
-    credentials: PropTypes.shape({
-        /** The email the user logged in with */
-        login: PropTypes.string,
-
-        /** The validate code */
-        validateCode: PropTypes.string,
-    }),
-
-    /** The details about the account that the user is signing in with */
-    account: PropTypes.shape({
-        /** Whether a sign on form is loading (being submitted) */
-        isLoading: PropTypes.bool,
-    }),
-};
-
-const defaultProps = {
-    route: validateLinkDefaultProps,
-    session: {
-        authToken: null,
+function ValidateLoginPage({
+    account,
+    credentials,
+    route: {
+        params: {accountID, validateCode},
```

Comment:
please make sure that `route.params` always exists

Reply:
@situchan It's expected to be defined every time the component is mounted 😄